src/macrofier/types.mjs

```diff
@@ -27,9 +27,11 @@ const primitives = {
   "integer": "number",
   "number": "number",
   "boolean": "boolean",
-  "string": "string"
+  "string": "string",
+  "float": "float"
```

Comment:
The keys in the map are JSON-Schema primitives. JSON-schema does not have a `float` type, so it should not be here.

Did you come across a schema w/ a `float` type? I'm surprised it passed validation.

Reply:
https://json-schema.org/understanding-json-schema/reference/numeric#:~:text=JSON%20does%20not%20have%20distinct,the%20same%20value%20in%20JSON.

Reply:
These are just default values for JavaScript. If you need different overrides for Cpp, they can go in the sdk.config.json

Reply:
yea, it is already overridden language config. I missed to remove. initially I did not clear idea about this setting. finally I understood and missed to remove it